xplique/plots/tabular.py

```diff
@@ -199,6 +199,7 @@ def plot_feature_impact(
 
     # add some text for labels and custom y-axis tick labels
     axes.set_xlabel('Impact on output')
```

Comment:
Is `X` really the impact on output ? I thought it was `X` percent (or 1 - percent) of data that are set to baseline state and `Y` for output score ?